sqlite.go

```diff
@@ -133,7 +133,7 @@ func (sw *SQLiteResultItemWriter) WriteRow(r any, written int) error {
 	for _, field := range sw.fields {
 		v := m[field]
 		switch t := v.(type) {
-		case map[string]any:
```

Comment:
I'm dropping this because converting maps to JSON isn't actually compatible with the old JSON style of ingestion. In that style all nested fields get expanded into separate columns but that's not what is happening here.